.github/ubuntu-22.04.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04
```

Comment:
If your dependency list is the same you can have one Dockerfile and pass in ubuntu version as an arg.
It is also ok to separate them like this if you want to change dependencies depending on ubuntu verison

Reply:
I've created an .sh script which will hold common setup code. So I've left it in two separate Dockerfiles in case we need to add some setup specific for one or another version.